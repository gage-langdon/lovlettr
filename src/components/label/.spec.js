import React from 'react';
import { shallow } from 'enzyme';

import Label from './index';

describe('Label Component', () => {
  it('Should render with default props', () => {
    const wrap = shallow(<Label>test</Label>);
    expect(wrap).toMatchSnapshot();
  });

  it('Should render with cursive font', () => {
    const wrap = shallow(<Label cursive>test</Label>);
    expect(wrap).toMatchSnapshot();
  });

  it('Should render with classNames as props', () => {
    const wrap = shallow(<Label className="test">test</Label>);
    expect(wrap).toMatchSnapshot();
  });

  it('Should render with a react component as the child', () => {
    const wrap = shallow(
      <Label className="test">
        <span>test</span>
      </Label>
    );
    expect(wrap).toMatchSnapshot();
  });
});
