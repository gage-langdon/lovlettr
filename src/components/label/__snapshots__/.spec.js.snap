// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Label Component Should render with classNames as props 1`] = `
<div
  className="label  test"
>
  test
</div>
`;

exports[`Label Component Should render with cursive font 1`] = `
<div
  className="label cursive "
>
  test
</div>
`;

exports[`Label Component Should render with default props 1`] = `
<div
  className="label  "
>
  test
</div>
`;
